src/main/java/io/stargate/sgv2/jsonapi/exception/APIException.java

```diff
@@ -97,11 +106,22 @@ public APIException(ErrorInstance errorInstance) {
     this.body = errorInstance.body();
     Objects.requireNonNull(errorInstance.httpStatusOverride(), "httpStatusOverride cannot be null");
     this.httpStatus = errorInstance.httpStatusOverride().orElse(DEFAULT_HTTP_STATUS);
+    Objects.requireNonNull(errorInstance.exceptionActions(), "exceptionActions cannot be null");
+    this.exceptionActions = errorInstance.exceptionActions();
   }
 
   public APIException(
```

Comment:
I know Javadoc comment was already missing, but if this is just for testing, maybe add a comment stating that? Otherwise it seems odd to have delegation working in this direction (from individual arguments to wrapper object)